shell/platform/android/io/flutter/embedding/engine/systemchannels/SettingsChannel.java

```diff
@@ -17,13 +24,30 @@ public class SettingsChannel {
   private static final String BRIEFLY_SHOW_PASSWORD = "brieflyShowPassword";
   private static final String ALWAYS_USE_24_HOUR_FORMAT = "alwaysUse24HourFormat";
   private static final String PLATFORM_BRIGHTNESS = "platformBrightness";
+  private static final String CONFIGURATION_ID = "configurationId";
+
+  // When hasNonlinearTextScalingSupport() returns false, this will not be initialized.
```

Comment:
Is this because Java will only initialize static fields on first access and when `hasNonlinearTextScalingSupport` is false this is never accessed?

Reply:
Yes.